GVFS/GVFS.Common/PlaceholderListDatabase.cs

```diff
@@ -165,12 +171,12 @@ public List<PlaceholderData> GetAllEntriesAndPrepToWriteAllEntries()
         ///     - If WriteAllEntriesAndFlush is *not* called entries that were added to the PlaceholderListDatabase after
         ///       calling GetAllEntriesAndPrepToWriteAllEntries will be lost

```

Comment:
Update method names in the comments (I see `GetAllEntriesAndPrepToWriteAllEntries` here, I have not checked if the other renamed methods were mentioned in comments)